build.gradle

```diff
@@ -114,6 +114,7 @@ project.ext.externalDependency = [
     'jsonSchemaAvro': 'com.github.fge:json-schema-avro:0.1.4',
     'jsonSimple': 'com.googlecode.json-simple:json-simple:1.1.1',
     'jsonSmart': 'net.minidev:json-smart:2.4.6',
+    'json': 'org.json:json:20090211',
```

Comment:
Minor: are we sure we need this new dependency? We aim to avoid adding these if possible since every dep can introduce new vulns

Reply:
yeah I needed `JSONObject` in order to convert a string json into a real object to do some checking and setting on the recipe. I tried with what was already in metadata-utils but I couldn't find anything.. but also this dependency isn't technically new as it's used in other java projects like datahub-graphql-core, but it was included in the `mixpanel` external dependency so it was never exposed as a top level external dep until now!